### PR TITLE
RJ_Fibonacci_search_algorithm.py

### PR DESCRIPTION
**Pull request by Richajaishwal0**

### CODE REVIEW

File: Fibonacci_search_algorithm
Comment (ShudarsanRegmi) [inline Fibonacci_search_algorithm:1]: Nice code! But I think you forgot the most important part: the .py extension.

**Review by ShudarsanRegmi — CHANGES_REQUESTED** [review]
Please make following changes:
- First name the file as XY(X=First letter of your first name, Y = First letter of your last name) followed by the name of the program, followed by file extension if you're rewriting the code which has already been done by others
- Keep track of iteration tables in a list. so that we can pass that to UI for rendering
-

**Review by TheShyamTripathi — COMMENTED** [review]
Use the table_printer.py from utils folder module to print the iteration table for each iteration. Make changes and resubmit it

**Review by TheShyamTripathi — APPROVED** [review]
The code implementation is good and can be merged.

File: Fibonacci_search_algorithm
Comment (Richajaishwal0) [reply Fibonacci_search_algorithm:1]: Can you pls delete this comment now ,

**Pull request merged**